src/Codata/Sized/Conat/Properties.agda

```diff
@@ -8,15 +8,16 @@
 
 module Codata.Sized.Conat.Properties where
 
-open import Size using (Size)
-open import Data.Nat.Base using (ℕ; zero; suc)
-open import Codata.Sized.Thunk using (Thunk; Thunk^R; force)
 open import Codata.Sized.Conat
 open import Codata.Sized.Conat.Bisimilarity using (_⊢_≈_; refl; zero; suc)
+open import Codata.Sized.Thunk using (Thunk; Thunk^R; force)
+open import Data.Nat.Base using (ℕ; zero; suc)
 open import Function.Base using (_∋_)
-open import Relation.Nullary.Decidable using (yes; no; map′)
-open import Relation.Nullary.Negation.Core using (¬_)
 open import Relation.Binary.Definitions using (Decidable)
+open import Relation.Nullary.Decidable using (map′)
+open import Relation.Nullary.Decidable.Core using (yes; no)
```

Comment:
```suggestion
open import Relation.Nullary.Decidable.Core using (yes; no; map′)
```
Not sure why you separated these out? `map` is defined in `Decidable`, but `map′` already in `Decidable.Core`.